service-front/module/Application/src/Controller/DonorFlowController.php

```diff
@@ -38,6 +38,7 @@ public function howWillDonorConfirmAction(): ViewModel|Response
         $optionsdata = $this->config['opg_settings']['identity_methods'];
         $detailsData = $this->opgApiService->getDetailsData($uuid);
 
+        $view->setVariable('date_sub_form', $dateSubForm);
```

Comment:
Is this supposed to be here?

Reply:
Yes - there's now a passport date check on the first page